Update config.py

keithy opened the pull request:
Correction to error in the schema

keithy committed:
Update config.py

Correction to error in the schema

dkatavic reviewed (approved):
Looks good, thanks @keithy  for contributing!

dkatavic committed:
feat: swift name option support (#161)

dkatavic committed:
Bump version: 1.2.1 → 1.2.2 (#162)

* Bump version: 1.2.1 → 1.2.2

* chore: add docs about pushing tags

BartoszBlizniak committed:
ceng-163: Fix repository pagination (#163)

* Fix repository pagination

BartoszBlizniak committed:
Create zipapp and upload to github release (#164)

BartoszBlizniak committed:
chore(add-v-to-tag) (#165)

Add `v` in front of the version to grab the correct tag version.

Bump version: 1.2.4 → 1.2.5

Bump version: 1.2.3 → 1.2.4

Bump version: 1.2.2 → 1.2.3

chrisimcevoy committed:
chore(eng-4230): update changelog (#166)

AndrewSpeed committed:
Update CLI to store access and refresh tokens (#169)

* Update direnv config to create .env if it doesn't exist

* Install python version using pyenv if not present

* Add keyring dependency

* Add utils for storing & retrieving access and refresh tokens for SSO use

AndrewSpeed committed:
Add support for SAML authentication (#174)

* WIP: Able to successfully perform SAML auth with & without 2FA and store tokens in keyring

* Hide TOTP input

* Ensure headers exist before trying to set value

* Pass Authorization header to ensure generated JWT token is valid

* WIP: Update cloudsmith-api to support JWT auth

* Downgrade cloudsmith-api

* Minor refactor & adding tests for access token app init logic

* Add tests for keyring

* Fix failing API init test

* Add SAML utils tests

* Remove TODO comment

* minor keyring refactor

* Minor webserver exception handling refactor

* Extract api host from api config to ensure we always have a value present

* Bind only to localhost

* Update README with auth command

* Add warning message when access token refresh fails, update fallback authentication error message to include cloudsmith auth

AndrewSpeed committed:
Update changelog with 'cloudsmith auth' command entry (#175)

AndrewSpeed committed:
Update version to 1.3.0 (#176)

AndrewSpeed committed:
Add keyring to setup.py (#177)

AndrewSpeed committed:
Bump CLI version to 1.3.1 (#178)

AndrewSpeed committed:
Remove keyring from requirements.in (#179)

chrisimcevoy committed:
feat(eng-6386): drop support for python 3.8 (#173)

* feat: drop support for python 3.8

* chore: add python3.13 to ci

* chore: rectify mock assertions in >= 3.12

* chore: add to unreleased section of changelog

chrisimcevoy committed:
Bump version: 1.3.1 → 1.4.0 (#180)

* chore: bring bumpversion.cfg back into sync with current release

* Bump version: 1.3.1 → 1.4.0

* chore: update changelog

chrisimcevoy committed:
fix(eng-6046): update cloudsmith-api to v2.0.16 (#181)

chrisimcevoy committed:
Bump version: 1.4.0 → 1.4.1 (#182)

* Bump version: 1.4.0 → 1.4.1

* chore: update changelog

BartoszBlizniak committed:
Merge branch 'master' into schema-correction

BartoszBlizniak committed:
CENG-450: Fix `cloudsmith auth` problem when user passed owner (#183)

* fix cloudsmith auth command to convert string correctly from user input

BartoszBlizniak committed:
CENG-452: Add --sort flag for package list command (#185)

BartoszBlizniak committed:
Bump version: 1.4.1 → 1.5.0 (#187)

cloudsmith-iduffy committed:
feat: add HTML templates for authentication success and error, and improve session handling in SSO (#189)

cloudsmith-iduffy committed:
Bump version: 1.5.0 → 1.6.2 (#191)

nicholasmccrea committed:
feat: Support upload of extra files (#190)

* feat: Support upload of extra files

* Improve help text description.

* Bump cli version. Update changelog. Bump api version.

* Fix typo

jmccayCS committed:
fix(ENG-7593): Update CLI so login command can handle 2FA verification (#188)

Changes:
- Add 2FA auth requirement when applicable for logins via the CLI.

jmccayCS committed:
chore: Update changelog for 1.7.1 (#195)

* Add entry to changelog.md

* Bump version

jmccayCS committed:
chore: Fix suppressed error detail (#196)

jmccayCS committed:
Bump version number (#197)

mhutchings-182 committed:
No ticket/remove cc (#198)

* Remove code climate from circleCI

* Remove code climate from circleCI

LowzG committed:
Support for managing User API Tokens via SAML authentication (#192)

* Add BaseApi class

* Add list_user_tokens() and refresh_user_token()

* Add tokens click commands

* Add create_user_token_saml()

* Refactor config

* Add feature to create API tokens via auth (saml/sso login)

* Ask user input for refreshing token

* docstring for new_config_messaging()

* Make token_slug optional

* Update error handling

* Fix error handling

* Update cloudsmith-api version

* Update code to use the new api bindings

* Add test for tokens

* fix failing required steps

* PR comments

* PR comments

* Update README

---------

Co-authored-by: Martin Hutchings <mhutchings@cloudsmith.io>

mhutchings-182 committed:
Eng 8099/cli tokens release (#199)

* Bump version: 1.7.2 → 1.8.0

* Update change log and bump version

lskillen committed:
Merge branch 'master' into schema-correction
